tests/Hedgehog.Tests/PropertyTests.fs

```diff
@@ -97,6 +97,36 @@ let propertyTests = testList "Property tests" [
                 count =! 1
                 //render.Contains "actual: 1" =! true // comment out for now since it causes the Fable test to fail
 
+
+    testCase "recheck passes after code is fixed" <| fun () ->
+        let expectedShrunkenInput = 1
+        let mutable target = 0
+        let prop =
+            property {
+                let! i = Range.constant 0 1_000_000 |> Gen.int32
```

Comment:
Since the range starts with `0`, can't it generate `i = 0` which would be `= target`?